src/components/complete-profile/CompleteProfileBlock.tsx

```diff
@@ -53,7 +60,7 @@ const CompleteProfileBlock: FC<CompleteProfileBlockProps> = ({
       OfferService.getUsersOffers({
```

Comment:
you don't have to use the useCallback hook at all, you can pass the `OfferService.getUsersOffers` function to the useAxios hook, or at least you should delete `isOfferCreated` from the dependancy array.

Reply:
I use isOfferCreated dependency because it makes the hook re-send request to get user offers when the new offer is created in CompleteProfileBlock. It was the main task) without this dependency the first created offer won’t be shown in CompleteProfileBlock unless the user refreshes the page

Reply:
ok, now I understand your idea, but we should use the `useEffect` hook for such purposes because it will make code much more readable

Reply:
ok, now I understand your idea, but we should use the `useEffect` hook if you want to re-send request because it will make code much more readable